front/app/modules/commercial/insights/admin/components/ShowHiddenNodes/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import { colors } from 'utils/styleUtils';
+
+// styles
+import styled from 'styled-components';
+
+// components
+import { Box, Icon } from '@citizenlab/cl2-component-library';
+import { TooltipContentList } from 'modules/commercial/insights/admin/components/StyledTextComponents';
+import Tippy from '@tippyjs/react';
+
+// intl
+
+import { FormattedMessage } from 'react-intl';
+import messages from '../../containers/Insights/messages';
+
+const TooltipIcon = styled(Icon)`
+  width: 17px;
+  height: 17px;
+  fill: ${colors.label};
+  margin: 0 5px 3px 0;
+`;
+
+export type NodeName = {
+  id: string;
+  name: string;
+};
+
+type ShowHiddenNodesProps = {
+  hiddenNodes: string[];
+  nodesNames: NodeName[] | false;
+  handleShowHiddenNodesClick: () => void;
+};
+
+const ShowHiddenNodes = ({
+  hiddenNodes,
+  nodesNames,
+  handleShowHiddenNodesClick,
+}: ShowHiddenNodesProps) => {
+  if (isNilOrError(hiddenNodes) || hiddenNodes.length === 0 || !nodesNames) {
+    return null;
+  }
+  const hiddenNodesNames = hiddenNodes.map((nodeId) =>
+    nodesNames.find((node: NodeName) => node.id === nodeId)
+  );
+  const hiddenNodesCount = hiddenNodes.length;
+  return (
+    <Box
+      mt="12px"
+      style={{
```

Comment:
We don't normally pass styles inline. Also - not a good idea to have onClick on a div - not very semantic. What you probably need here is a button of some kind?